### PR TITLE
Install s2i if not already installed

### PR DESCRIPTION
**Pull request by Divyansh42**
Signed-off-by: divyansh42 <diagrawa@redhat.com>

### Related Issue(s)
Closes: #33 

### Checklist

- [x] This PR includes a documentation change
- [ ] This PR does not need a documentation change
---
- [x] This PR includes test changes
- [ ] This PR's changes are already tested
---
- [ ] This change is a patch change
- [x] This change is a minor change
- [ ] This change is a major (breaking) change

### CODE REVIEW

**Review by tetchel — CHANGES_REQUESTED** [review]
this looks good, let's exec 's2i version' at the top of the action like we do with podman and buildah

**Comment by Divyansh42** [comment]
> this looks good, let's exec 's2i version' at the top of the action like we do with podman and buildah

Actually, I removed this as `s2i` itself prints the version when we do `s2i build....`
So it's a kind of redundant information here.

File: README.md [hunk 1]
```diff
@@ -16,16 +16,19 @@ and letting the container prepare that source code for execution. The base
...
 
+To install any specific version of `s2i` use [Openshift Tools Installer](https://github.com/marketplace/actions/openshift-client-installer).
+
 **NOTE:**
 `s2i-build` only works on Linux platforms, because it relies on the Docker daemon.<br>
 If you are using GitHub's Ubuntu runners, the Docker daemon will already be available.
 Otherwise, you can use [Docker Setup Buildx](https://github.com/marketplace/actions/docker-setup-buildx) to set up and start the Docker daemon.
 
-Also see [buildah-build](https://github.com/redhat-actions/buildah-build) for more configurable method of building images, from scratch or from a Dockerfile.
+Also see [Buildah Build](https://github.com/marketplace/actions/buildah-build) for more configurable method of building images, from scratch or from a Dockerfile.
```
Comment (tetchel) [inline README.md:29]: let us bold these, and write them in their slug forms.

```
also see [**buildah-build**]
```

File: README.md [hunk 2]
```diff
@@ -16,16 +16,19 @@ and letting the container prepare that source code for execution. The base
 S2I container images contain the language runtime and build tools needed for
 building and running the source code.
 
+To install `s2i` use [Openshift Tools Installer]
```
Comment (tetchel) [inline README.md:19]: delete this?

Comment (Divyansh42) [reply README.md:19]: Oops...Sorry

File: README.md (hunk 1 above)
Comment (Divyansh42) [reply README.md:29]: Okay

**Review by tetchel — APPROVED** [review]

**Pull request merged**